src/main/java/teammates/ui/controller/InstructorCourseDetailsPageData.java

```diff
@@ -20,11 +24,148 @@ public InstructorCourseDetailsPageData(AccountAttributes account) {
 
     public InstructorAttributes currentInstructor;
     public CourseDetailsBundle courseDetails;
-    public List<StudentAttributes> students;
     public List<InstructorAttributes> instructors;
     public String studentListHtmlTableAsString;
+    public ElementTag giveCommentButton;
+    public ElementTag courseRemindButton;
+    public CourseDetailsStudentsTable studentsTable;
     
-    public String getInstructorCourseRemindLink() {
+    public void init(InstructorAttributes currentInstructor, CourseDetailsBundle courseDetails, 
+                     List<InstructorAttributes> instructors, List<StudentAttributes> students) {
+        this.currentInstructor = currentInstructor;
+        this.courseDetails = courseDetails;
+        this.instructors = instructors;
+        
+        boolean isDisabled = !currentInstructor.isAllowedForPrivilege(
+                                                    Const.ParamsNames.INSTRUCTOR_PERMISSION_GIVE_COMMENT_IN_SECTIONS);
+        
+        String content = "<span class=\"glyphicon glyphicon-comment glyphicon-primary\"></span>";
+        giveCommentButton = createButton(content, "btn btn-default btn-xs icon-button pull-right", 
+                                         "button_add_comment", null, "", "tooltip", null, isDisabled);
+        
+        isDisabled = !currentInstructor.isAllowedForPrivilege(Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_STUDENT);
+        String onClick = "if(toggleSendRegistrationKeysConfirmation('" 
+                          + Sanitizer.sanitizeForJs(courseDetails.course.id) + "')) "
+                          + "window.location.href='" 
+                          + Sanitizer.sanitizeForJs(getInstructorCourseRemindLink()) + "';";
```

Comment:
No need for `Sanitizer.` when you're extending from `PageData`.
This seems to be inconsistently done in this file, by the way (the calls bellow don't have `Sanitizer.`, but these do).